reporter/src/funTest/assets/file-counter-expected-scan-report.html

```diff
@@ -255,7 +255,7 @@
             <body>
             <div id="report-container">
                   <div class="ort-report-label">Scan Report</div>
-                <h2>Index</h2><ul><li><a href="#license-check-results">License Check Results (0 errors)</a></li><li><a href="#error-summary">Error Summary (1)</a></li><li><a href="#Gradle:com.here.ort.gradle.example:lib:1.0.0">Gradle:com.here.ort.gradle.example:lib:1.0.0</a></li></ul><h2><a id="license-check-results"></a>License Check Results</h2>No issues found.<h2><a id="error-summary"></a>Error Summary (1)</h2>
+                <h2>Index</h2><ul><li><a href="#policy-violation-summary">Policy Violation Summary (0)</a></li><li><a href="#error-summary">Error Summary (1)</a></li><li><a href="#Gradle:com.here.ort.gradle.example:lib:1.0.0">Gradle:com.here.ort.gradle.example:lib:1.0.0</a></li></ul><h2><a id="policy-violation-summary"></a>Policy Violation Summary</h2>No issues found.<h2><a id="error-summary"></a>Error Summary (1)</h2>
```

Comment:
Why not `(0 violations)`? `(0)` could be confusing.

Reply:
I like '0 violations' more than just '0'.
I'd prefer to keep this commit as is as I think this change improves the situation.
Adding the 'violations' I would thus prefer to do in a commit on top. Is this fine with you?